transport.validator.mapcss

```diff
@@ -149,6 +149,19 @@ relation.pt_route[tag(route)!=parent_tag(route_master)]
 	throwError: tr("The public transport mode should be the same for the route and the route_master : {0} vs {1}", tag(route), parent_tag(route_master));
 }
 
+relation.pt_route[colour][colour=~/^#/][colour!~/^#([0-9A-Fa-f]{3}|[0-9A-Fa-f]{6})$/],
+relation.pt_route_master[colour][colour=~/^#/][colour!~/^#([0-9A-Fa-f]{3}|[0-9A-Fa-f]{6})$/]
+{
+    throwError: tr("Check the colour tag");
+}
+
+relation.pt_route[!colour][color],
+relation.pt_route_master[!colour][color]
+{
+    throwError: tr("The color of the public transport line should be in a colour tag");
+    fixChangeKey: "color=>colour";
+}
+
```

Comment:
There is already a more detailed mapcss check for `colour`. May be we can use and promote this one in place:
https://github.com/osm-fr/osmose-backend/blob/master/plugins/Colour.validator.mapcss

Reply:
Fun fact that the existing Mapcss file for colour codes does not check the actual `colour` tag! but sure, I can add it in this file.

But I'm considering to add it to the transport mapcss anyway:
- the colour mapcss file is not easy to add to JOSM. Most transport contributors won't add it at all
- a specific item/class in Osmose (or a tag, or some other way to filter them in the API) would be cool so this check can be considered in transport specific QA applications (such as Bifidus or custom scripts used by Jungle Bus on their missions). I want to be able to get these colour errors on pt lines, but I don't want to be pulluted by the colour erros on buildings. 

What do you think ? How bad would it be to have these errors duplicated in Osmose ?

Reply:
@frodrigo : would `-osmoseItemClassLevel: none;` do the trick to have the test in mapcss but not in Osmose (as it already exists in another form) ?

Reply:
Yes `osmoseItemClassLevel: none;` rules are filtered out.

Reply:
Thanks!